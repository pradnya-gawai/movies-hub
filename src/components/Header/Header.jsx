import React from "react";
import "./Header.css";

function Header() {
  return (
    <div>
      <span className="header" data-testid="header">Movies Hub </span>
    </div>
  );
}

export default Header;
