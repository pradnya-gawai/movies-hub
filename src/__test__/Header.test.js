import { render, screen } from "@testing-library/react";
import Header from "../components/Header/Header";
import renderer from 'react-test-renderer';

describe("Header", () => {
  it("should render property", () => {
    render(<Header />);
    const headerElement = screen.getByText(/movies hub/i);
    expect(headerElement).toBeInTheDocument();
  });
});
it('"snapshot" of header component', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
