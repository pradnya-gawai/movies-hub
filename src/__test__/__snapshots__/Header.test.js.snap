// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"snapshot" of header component 1`] = `
<div>
  <span
    className="header"
  >
    ðŸŽ¬ Movies Hub ðŸŽ¬
  </span>
</div>
`;
